packages/web-react/src/components/Card/CardMedia.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+
+import classNames from 'classnames';
+import React from 'react';
+import { useStyleProps } from '../../hooks';
+import { CardSizes, SpiritCardMediaProps } from '../../types';
+import { useCardStyleProps } from './useCardStyleProps';
+
+const defaultProps: Partial<SpiritCardMediaProps> = {
+  hasFilledHeight: false,
+  isExpanded: false,
+  size: CardSizes.AUTO,
+};
+
+const CardMedia = (props: SpiritCardMediaProps) => {
+  const propsWithDefaults = { ...defaultProps, ...props };
+  const { children, size, isExpanded, hasFilledHeight, ...restProps } = propsWithDefaults;
+  const { classProps } = useCardStyleProps({ size, isExpanded, hasFilledHeight });
+  const { styleProps, props: otherProps } = useStyleProps(restProps);
+
+  return (
+    <div {...otherProps} className={classNames(classProps.media, styleProps.className)} style={styleProps.style}>
+      <div className="CardMedia__canvas">{children}</div>
```

Comment:
Please move this class to the `useCardStyleProps` hook too, because otherwise it won't be prefixed if they turn it on.